### ISSUE TITLE
False positive for `.toString`

### ISSUE THREAD

**Issue by oziniak** [mattorchard/eslint-plugin-require-explicit-generics#4]
Hi, nice plugin!

Currently, I observe an error for every `.toString` in my TS React project with the following configuration:
```
    'require-explicit-generics/require-explicit-generics': [
      'error',
      ['useState']
    ]
```
This helps to remove errors but feels like a hack.

```
    'require-explicit-generics/require-explicit-generics': [
      'error',
      {
        toString: 0,
        useState: 1
      }
    ]
```
Glancing through the code, I see no `toString` mentions.

**Comment by mattorchard** [comment]
Howdy, thank you for creating a ticket!

I was able to recreate the issue, so I'll try to find out what's going wrong later this week.

**Comment by mattorchard** [comment]
Ahh I found the source of the bug: the use of: `expectedCountMap[possibleName]` will return true on anything in the object prototype (such as `toString`)

I have a fix drafted, and will do some more testing later in the week before releasing an update

**Comment by mattorchard** [comment]
Just published a fix with version `0.4.2`

@oziniak please let me know if you can remove the workaround now.

**Comment by oziniak** [comment]
@mattorchard works like a charm, thanks
